src/main/java/com/avispl/symphony/dal/infrastructure/management/neat/pulse/NeatPulseCommunicator.java

```diff
@@ -233,6 +229,61 @@ public void stop() {
 		}
 	}
 
+	/**
+	 * Get list devices sensor information
+	 */
+	private void retrieveDeviceSensorInformation() {
+		try {
+			JsonNode response = this.doGet(String.format(NeatPulseCommand.LIST_DEVICE_SENSOR, this.getLogin()), JsonNode.class);
+			if (response != null && response.has(NeatPulseConstant.DATA)) {
+				for (JsonNode dataNode : response.get(NeatPulseConstant.DATA)) {
+					String id = dataNode.get("id").asText();
+					JsonNode endpointData = dataNode.get("endpointData");
+
+					if (endpointData != null && endpointData.has("data")) {
+						JsonNode sensorData = endpointData.get("data");
+						mapOfDeviceIdAndDeviceSensor.put(id, sensorData);
+					}
+				}
+			}
+		} catch (CommandFailureException ex) {
+			// Device not support the sensor command
+			logger.info("Device not support the sensor command", ex);
```

Comment:
probably should be "warn" level, having an unsupported command addressed would mean something went south prior to this point